src/test/java/dk/alexandra/fresco/lib/arithmetic/BasicArithmeticTests.java

```diff
@@ -553,4 +573,362 @@ public ProtocolProducer prepareApplication(
 		}
 	}
 	
+	/**
```

Comment:
In general for all the new tests - I think they should be moved to a, say, StatisticsTest class instead.